libbpf-tools/Makefile

```diff
@@ -40,6 +40,7 @@ APPS = \
 	ksnoop \
 	llcstat \
 	mountsnoop \
+	memleak \
```

Comment:
Please keep this list sorted and add an entry to `.gitignore`.

Reply:
Fix in commit [2beb289e](https://github.com/iovisor/bcc/pull/3828/commits/2beb289e98d651bddc2127299c5627b5d6149fce).